seacrowd/sea_datasets/thai_romanization/thai_romanization.py

```diff
@@ -0,0 +1,190 @@
+# coding=utf-8
+# Copyright 2022 The HuggingFace Datasets Authors and the current dataset script contributor.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""
+The Thai Romanization dataset contains 648,241 Thai words \
+    that were transliterated into English, making Thai \
+    pronounciation easier for non-native Thai speakers. \
+    This is a valuable dataset for Thai language learners \
+    and researchers working on Thai language processing task. \
+    Each word in the Thai Romanization dataset is paired with \
+    its English phonetic representation, enabling accurate \
+    pronunciation guidance. This facilitates the learning and \
+    practice of Thai pronunciation for individuals who may not \
+    be familiar with the Thai script. The dataset aids in improving \
+    the accessibility and usability of Thai language resources, \
+    supporting applications such as speech recognition, text-to-speech \
+    synthesis, and machine translation. It enables the development of \
+    Thai language tools that can benefit Thai learners, tourists, \
+    and those interested in Thai culture and language.
+"""
+import os
+from pathlib import Path
+from typing import Dict, List, Tuple
+
+import datasets
+import pandas as pd
+
+from seacrowd.utils import schemas
+from seacrowd.utils.configs import SEACrowdConfig
+from seacrowd.utils.constants import Tasks, Licenses
+
+# There are no citation available for this dataset.
+_CITATION = """\
+@article{,
+  author    = {},
+  title     = {},
+  journal   = {},
+  volume    = {},
+  year      = {},
+  url       = {},
+  doi       = {},
+  biburl    = {},
+  bibsource = {}
+}
```

Comment:
Maybe make this an empty string or `None`?

Reply:
Solved on https://github.com/SEACrowd/seacrowd-datahub/pull/645/commits/805a0fb0c92e7284e38d99571bc1c5fdefea9451 commit, Sir.